tests/integration/kubernetes/gha-run.sh

```diff
@@ -126,7 +126,7 @@ function deploy_kata() {
 	# Enable 'default_vcpus' hypervisor annotation
 	yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[6].value' "default_vcpus"
 
-	if [ "${KATA_HOST_OS}" = "cbl-mariner" ]; then
+	if [ "${KATA_HOST_OS}" = "cbl-mariner" ] || [ "${KATA_HOST_OS}" = "ubuntu" ]; then
```

Comment:
@GabyCT, I don't think we want to do it here.

What I think we should do is creating a new check, after this one, like:
```sh
if [ "${KATA_HYPERVISOR}" = "qemu" ]; then
    yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[6].value' "image initrd kernel default_vcpus"
fi
```